experiments/modern/src/maki-interpreter/constants.js

```diff
@@ -13,10 +13,10 @@ const COMMANDS = {
   },
   "8": { name: "eq", short: "==", in: "2", out: "1" },
   "9": { name: "heq", short: "!=", in: "2", out: "1" },
-  "10": { name: "le", short: "<", in: "2", out: "1" },
```

Comment:
I double and triple checked, but these seem to be wrong? Based on the order of the tests in the basicTests

Reply:
Interesting. You can confirm by running the decompiler on a script and seeing if it swaps the greater than/less than symbols.

Reply:
Yea, I wonder if its just like a variable ordering thing or what, but yea, I should test it out in the decompiler to see how it handles it.